server/odc-service/src/main/java/com/oceanbase/odc/service/connection/model/ConnectionConfig.java

```diff
@@ -306,6 +306,11 @@ public class ConnectionConfig
     @JsonProperty(access = Access.READ_ONLY)
     private EnvironmentStyle environmentStyle;
 
+    private Long projectId;
```

Comment:
project is not related to datasource but database, I prefer not adding this field which will violate our design

Reply:
The second solution is more reasonable, I will refine it